nitro-protocol/contracts/ForceMove.sol

```diff
@@ -139,6 +139,7 @@ contract ForceMove is IForceMove, StatusManager {
 
         // checks
         _requireStateSupportedBy(fixedPart, signedVariableParts, channelId);
+        _requireEachStateFinal(signedVariableParts);
```

Comment:
We should consider the following, streamlined version of this:
1. require a single state
2. require that state to have `isFInal=true`
3. require `nParticipants` signatures
4. do not defer to the application rules at all

As it is, we have a slightly confusing soup of core logic vs application defined logic dictating whether `conclude` will succeed.

Reply:
[84f8376](https://github.com/statechannels/go-nitro/pull/814/commits/84f8376cd4adc3f2b79cec59d0940da1ca92a7c5)

Reply:
I have left a [comment](https://github.com/statechannels/go-nitro/issues/813#issuecomment-1201157704) describing what I believe we also have to bear in mind, while making these changes.